### PR TITLE
forget account button with confirmation. fixes #221

### LINKED ISSUE #221: Add a "forget account" button next to cached accounts in login page
This is useful for both the user and the developer. Basically:

* take the addr
* compute the folder name
* wipe the folder (needs confirmation dialog)

### PR DESCRIPTION
**Pull request by karissa**
screenshot from 2018-12-18 16-59-58

### CODE REVIEW

**Comment by Jikstra** [comment]
I get this error when i try to remove a used account of me:
```
{ Error: ENOTEMPTY: directory not empty, rmdir '/home/asd/.config/DeltaChat/64656c746174657374406d7261792e6465'
    at Object.fs.rmdirSync (fs.js:821:3)
    at EventEmitter.ipc.on (/home/kerle/Workspace/deltachat-desktop/src/main/ipc.js:67:8)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (/usr/lib/electron/resources/electron.asar/browser/api/web-contents.js:284:13)
    at WebContents.emit (events.js:182:13)
  errno: -39,
  syscall: 'rmdir',
  code: 'ENOTEMPTY',
  path:
   '/home/asd/.config/DeltaChat/64656c746174657374406d7261792e6465' }
```

**Comment by ralphtheninja** [comment]
Looks nice!

File: src/main/ipc.js [hunk 1]
```diff
@@ -55,6 +62,13 @@ function init (cwd, state) {
     dc.login(...args, render, txCoreStrings())
   })
 
+  ipc.on('forgetLogin', (e, addr) => {
+    var targetDir = dc.getPath(addr)
+    fs.rmdirSync(targetDir)
```
[the remark this answers is not in the document]
Comment (Jikstra) [reply src/main/ipc.js]: Oh i tried to replace it with rimraf, it then successfully deleted the folder but didn't render correctly (no account was shown).

[the remark this answers is not in the document]
Comment (karissa) [reply src/main/ipc.js]: isn't the account supposed to be removed from the UI too? I don't understand the issue.

File: src/main/deltachat.js [hunk 2]
```diff
@@ -26,12 +25,16 @@ class DeltaChatController extends events.EventEmitter {
...
+    return path.join(this.cwd, Buffer.from(addr).toString('hex'))
+  }
+
   /**
    * Dispatched when logging in from Login
    */
   login (credentials, render, coreStrings) {
     // Creates a separate DB file for each login
-    const cwd = path.join(this.cwd, Buffer.from(credentials.addr).toString('hex'))
+    const cwd = this.getPath(credentials.addr)
```
Comment (karissa) [inline src/main/deltachat.js]: this finally also makes src/main/deltachat.js entirely separate and independent from `electron` which gets us closer to the grand extension idea.. @hpk42

File: src/main/ipc.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jikstra) [reply src/main/ipc.js]: I have multiple accounts (4). I deleted one, after that none were shown anymore.

**Pull request merged**